.github/workflows/ci.yaml

```diff
@@ -1,6 +1,10 @@
 name: ci
 
-on: [push, pull_request]
+on:
+  push:
+    branches: [master, develop, develop-2.0]
```

Comment:
Why this change? Even I work on the other branches I'd also love to see the CI results.

Reply:
You can see the same CI result in the PR.
We only need to ensure that CI can run successfully on these three branches

Reply:
Nope, not every barnch leads to a PR, but I want to see the result.